client/package.json

```diff
@@ -4,9 +4,9 @@
   "description": "",
   "main": "webpack.config.js",
   "scripts": {
-    "test": "jest --coverage",
+    "test": "jest --colors --coverage",
     "type-check": "tsc --noEmit",
-    "start": "webpack serve --open --hot --mode development",
+    "start": "webpack serve --open --hot --mode=development",
```

Comment:
![itsme](https://github.com/Code-Hammers/code-hammers/assets/89617737/d01c0f7f-726e-4d72-a1a4-21027bc55dfb)

Saw in the webpack docs they set mode like this. Updated the build script too in a later commit. We could also remove `mode: process.env.NODE_ENV` in the webpack config since we're explicitly setting it via CLI

Reply:
latest commit removed mode from webpack config